modules/sdk/gradle-plugins-defaults/src/main/java/com/liferay/gradle/plugins/defaults/LiferayOSGiDefaultsPlugin.java

```diff
@@ -89,7 +89,9 @@
 import com.liferay.gradle.util.copy.RenameDependencyClosure;
 import com.liferay.gradle.util.copy.ReplaceLeadingPathAction;
 import com.liferay.gradle.util.copy.StripPathSegmentsAction;
+import com.liferay.portal.kernel.util.PropsKeys;
```

Comment:
@robertoDiaz The Gradle plugin does not have access to portal-kernel classes at compile time before your change and I believe this is on purpose. Please contact @PeterShin for help or review for the Gradle plugin part.

Reply:
Hi @dantewang , if you don't mind I'll use this PR to ask him. Are you ok with that?

Hey @petershin, 

Any thoughts about this?

Reply:
Well, In fact this is not a new pattern, just read and honor the existing configuration properties.

If I've understand it well I only have to change https://github.com/liferay/liferay-portal/blob/master/build-test.xml#L9612 and resend the PR to you.

Is it right?

Reply:
@robertoDiaz - It was just one example. Please update all the build-text-xxx.xml files in your pull.

Reply:
Got it!